src/Serilog.Sinks.Graylog/MessageBuilders/GelfMessageBuilder.cs

```
@@ -29,9 +27,9 @@ public class GelfMessageBuilder : IMessageBuilder
         /// <param name="options">The options.</param>
         public GelfMessageBuilder(string hostName, GraylogSinkOptions options)
         {
-            _hostName = hostName;
-            Options = options;
-            _propertyNamingStrategy = options.PropertyNamingStrategy;
+            Options = options ?? throw new ArgumentNullException(nameof(options));
```

Fail fast when dependencies are not provided. Should prefer this to an eventual NRE during log calls.